jobs/viasto_senior_python_engineer_08_2015.html

```diff
@@ -0,0 +1,54 @@
+---
+title: Senior Python Engineer 
+company: viasto GmbH
+url: https://www.viasto.com/en/
+location: Berlin, Germany
+contract: permanent
+contact:
+    name: Evelyne Quansah
+    email: evelyne.quansah@viasto.com
+created: !!timestamp '2015-02-20'
```

Comment:
You should update this date to the current date or your listing will be shown at the bottom!